sql/core/src/test/scala/org/apache/spark/sql/execution/arrow/ArrowConvertersSuite.scala

```diff
@@ -391,6 +392,85 @@ class ArrowConvertersSuite extends SharedSQLContext with BeforeAndAfterAll {
     collectAndValidate(df, json, "floating_point-double_precision.json")
   }
 
+  ignore("decimal conversion") {
```

Comment:
That might be true, I haven't looked into it yet.  I can work on adding support on the Arrow side, so I'll try to check on that and see where it stands in the upcoming 0.5 release.